pkgs/development/libraries/xxHash/default.nix

```diff
@@ -11,22 +11,17 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-2WoYCO6QRHWrbGP2mK04/sLNTyQLOuL3urVktilAwMA=";
   };
 
-  # Upstream Makefile does not anticipate that user may not want to
-  # build .so library.
-  postPatch = lib.optionalString stdenv.hostPlatform.isStatic ''
-    sed -i 's/lib: libxxhash.a libxxhash/lib: libxxhash.a/' Makefile
-    sed -i '/LIBXXH) $(DESTDIR/ d' Makefile
-  '';
-
-  outputs = [ "out" "dev" ];
-
-  makeFlags = [ "PREFIX=$(dev)" "EXEC_PREFIX=$(out)" ];
-
   # pkgs/build-support/setup-hooks/compress-man-pages.sh hook fails
   # to compress symlinked manpages. Avoid compressing manpages until
   # it's fixed.
   dontGzipMan = true;
```

Comment:
Can you check this?

Reply:
So actually, man pages didn't work at all without a patch, which I just added. Also, I made it build XXHSum by default, to replicate earlier behaviour.